TicTacToeBot/Headers/TreeNode.h

```diff
@@ -0,0 +1,40 @@
+//
+// Created by Sam Petrov on 22.04.2020.
+//
+#ifndef TICTACTOE_TREENODE_H
+#define TICTACTOE_TREENODE_H
+
+#include <vector>
+#include "PlayField.h"
+
+class TreeNode{
+public:
+    TreeNode(const PlayField f, TreeNode* p = nullptr): field(f), parent(p) {}
```

Comment:
Можно ещё так сделать
```suggestion
    TreeNode(const PlayField f = PlayField(), TreeNode* p = nullptr): field(f), parent(p) {}
```